server/services/odata.service.js

```diff
@@ -256,12 +294,17 @@ module.exports = class ODataService {
 
       console.log(`Patching URL: [${url}]`)
```

Comment:
There shouldn't be `console.log` commands in the code - this should be removed

(I think this line had been left in erroneously from earlier development)